pkg/lib/cockpit-po-plugin.js

```diff
@@ -88,7 +88,7 @@ module.exports = class {
                 for (const [msgid, translation] of Object.entries(context)) {
                     /* Only include msgids which appear in this source directory */
                     const references = translation.comments.reference.split(/\s/);
-                    if (!references.some(str => str.startsWith(`pkg/${this.subdir}`)))
+                    if (!references.some(str => str.startsWith(`pkg/${this.subdir}`) || str.startsWith('src')))
```

Comment:
This seems wrong to me, we want to include references from `src` for plugins. With this change we end up skipping translations from src. I wonder how this used to work as it required less magic in the past?

Reply:
We don't skip. see the condition again: `if (!maching_pattern) continue`

Reply:
Right, I thought it was outside of the `.some`...